src/PleskX/Api/Operator/SiteAlias.php

```diff
@@ -32,4 +32,20 @@ public function create(array $properties, array $preferences = [])
         return new Struct\Info($response);
     }
 
+	/**
+	 * @param string $siteName
+	 * @return Struct\Info
+	 */
+	public function delete(string $siteName)
```

Comment:
I'd prefer the interface with `$field, $value` in order to be able to delete an object by id.